en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -294,6 +379,143 @@ Run any of the following CTL commands to get keys for the API/API Product.
+
+    ```bash
+    $ apictl export-app --name PetsApp --owner david -e dev --withKeys
+
+    Successfully exported Application!
+    Find the exported Application at /home/wso2user/.wso2apictl/exported/apps/dev/david_PetsApp.zip
+    ```
+
+2.  Extract the exported Application Project.
+3.  Commit the project to the same git repository.
+4.  Run `vcs status` command to see the available changes that needs to be deployed to the production environment.
+
+    !!! example
+        ```bash
+        $ apictl vcs status -e prod
+        Projects to Deploy (1)
+
+        Applications (1) ...
+        1: [save]		PetsApp: (PetsApp)
+        ```
+
+3.  Import the **PetsApp** Application into the production environment by running the following sample command.
+
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Deploying Projects (1)...
+
+        Applications (1) ...
+        1: PetsApp: (PetsApp)
+        Successfully imported Application
+        ```
+
+    Now the tool will automatically detect the target environment and create the application in the target environment.
+
+    !!! important
+        For deploying an application using `vcs deploy` command:
+        
+        -   It is mandatory to have your Application projects in a GIT based version control system.
+        -   It is mandatory to have `application_params.yaml` file inside each application project. This is created by default when you exported an Application using `export-app`. In `application_params.yaml`. The below configuration section is used to deploy the application.
+
+        ```bash
+        vcs:
+            import:
+                update: true
+                preserveOwner: true
+                skipSubscriptions: false
+                skipKeys: true
+        ```
+
+        You can change the above fields accordingly.
+
+        | Field             | Description                                                                                 |
+        |-------------------|---------------------------------------------------------------------------------------------|
+        | update            | Used to specify whether to update the application if it already exists during the deployment|
+        | preserveOwner     | Preserve existing owner of the application after importing it                               |
+        | skipSubscriptions | Specifies whether to import the subscriptions of the application                            |
+        | skipKeys          | Specifies whether to import the credentials of the application                              |
+
+    !!! tip
+        If you have multiple applications to promote through CI/CD, you can commit those applications projects as well to the same repository similar to the APIs and API Products. The rest of the process will be the same as above.
```

Comment:
Multiple applications could be promoted through CI/CD by committing those application projects to the same repository.